custom.props

```diff
@@ -3,11 +3,10 @@
   <PropertyGroup Label="Version">
     <VersionMajor>3</VersionMajor>
     <VersionMinor>2</VersionMinor>
-	<VersionPatch>1</VersionPatch>
```

Comment:
Why'd you remove the patch?

Reply:
I've added it :)
UWP renderer/object model won't rely on this file anymore, so no need for it